lib/healthlocker.ex

```diff
@@ -14,6 +14,7 @@ defmodule Healthlocker do
       supervisor(Healthlocker.Endpoint, []),
       # Start your own worker by calling: Healthlocker.Worker.start_link(arg1, arg2, arg3)
       # worker(Healthlocker.Worker, [arg1, arg2, arg3]),
+      worker(Segment, [Application.get_env(:segment, :write_key)])
```

Comment:
I came across `System.get_env` and `Application.get_env` when I was making the feedback feature. Can you explain the difference/when to use which? I didn't fully understand from the docs. I had to use System.get_env for what I was doing and couldn't get Application.get_env to work, so I wasn't sure if there was a specific case to use this.